rskj-core/src/test/java/co/rsk/core/BlockHeaderTest.java

```diff
@@ -136,7 +136,7 @@ public void getMiningForkDetectionDataNoDataCanBeFound() {
         header.setBitcoinMergedMiningCoinbaseTransaction(coinbase);
         header.seal();
 
-        assertThat(new byte[12], is(header.getMiningForkDetectionData()));
+        header.getMiningForkDetectionData();
```

Comment:
is this intentional?

Reply:
It is because the handling of that case changed from returning array to throwing an exception